File: JavaScript/Q15.js
let char = '$';
if ((char >= 'a' && char <= 'z') || (char >= 'A' && char <= 'Z')) {
  console.log("The character is an alphabet.");
} else if (char >= '0' && char <= '9') {
  console.log("The character is a digit.");
} else {
  console.log("The character is a special character.");
}

File: Java/Q15.java
class Main {
    public static void main(String[] args) {
        char ch = '$';
        if (Character.isLetter(ch)) {
            System.out.println("The character is an alphabet.");
        } else if (Character.isDigit(ch)) {
            System.out.println("The character is a digit.");
        } else {
            System.out.println("The character is a special character.");
        }
    }
}